issue fixed #20221 add your review text is not show uniformly in Mobi…

cedarvinda opened the pull request:
issue fixed #20221 add your review text is not show uniformly in Mobile view

### Manual testing scenarios (*)

1. Chrome Browser
2. Mobile View
3. Home Page
4. Product listing page

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

cedarvinda committed:
issue fixed #20221 add your review text is not show uniformly in Mobile view

issue fixed #20221 add your review text is not show uniformly in Mobile view

magento-engcom-team commented:
Hi @cedarvinda. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

VladimirZaets commented on app/design/frontend/Magento/luma/Magento_Review/web/css/source/_module.less:
```diff
@@ -393,6 +393,21 @@
+    .product-reviews-summary{
+        .reviews-actions {
+            a{
+                display: block;
```
Hi @cedarvinda. Thanks for collaboration. Please add space after `a` declaration.

cedarvinda replied:
@VladimirZaets ok, Thanks

cedarvinda committed:
_module.less updated for this issue ( add your review text is not show uniformly in Mobile view )

_module.less updated for this issue ( add your review text is not show uniformly in Mobile view )

VladimirZaets commented:
@cedarvinda thanks

magento-engcom-team commented:
Hi @VladimirZaets, thank you for the review.
[ENGCOM-3904](https://jira.corp.magento.com/browse/ENGCOM-3904) has been created to process this Pull Request

VladimirZaets commented:
Hi @cedarvinda. Thanks for collaboration. 
This issue was fixed in scope of this [PR](https://github.com/magento/magento2/pull/20224)
